src/components/bestSellers/bestSellers.styled.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import OpenBook from './photos/open-book-800px.jpg'
+// 400px x 200px
+// 600px x 300px
+// 800px x 400px
+
+export const MainWindow = styled.div`
+    background-image: url(${OpenBook});
```

Comment:
jeżeli chcesz różny rozmiar bg w zależności od szerokości okna to może ta wartość w mediaqueries ?